src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -98,7 +98,7 @@ case class WorkflowDescriptor(id: WorkflowId,
     def copyFile(file: WdlFile): Try[Unit] = {
       val src = file.valueString.toPath(gcsFilesystem)
       val wfPath = wfContext.root.toPath(gcsFilesystem).toAbsolutePath // Absolute path of the workflow directory
-      val relativeFilePath = src.subpath(wfPath.getNameCount, src.getNameCount) // Path of the src file relatively to the workflow directory
+      val relativeFilePath = src.subpath(wfPath.getNameCount, src.getNameCount) // Path of the src file relative to the workflow directory
```

Comment:
I think I said this in @Horneth PR but I think these comments are completely superfluous due to the name and the brevity of the line